### PR TITLE
Misc. fixups

### PR DESCRIPTION
**Pull request by GraionDilach**
Some misc stuff I did today.

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -226,9 +226,11 @@ MINER2:
 		Sequence: active
 	UpdatesDerrickCount:
 	Building:
+		Footprint: xx xx xx
+		Dimensions: 2,3
 		RequiresBaseProvider: false
-		TerrainTypes: Ore
-		AllowInvalidPlacement: true # TODO allow placement only on ressources
+		TerrainTypes: Clear, Road, Crater, Grass, Ore
+		AllowInvalidPlacement: true # TODO CanPlaceBuilding explicitly prevents placing upon tiles with resources
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml]: Yeah I know. I plan to patch that.

File: mods/hv/rules/buildings.yaml [hunk 2]
```diff
@@ -226,9 +226,11 @@ MINER2:
 		Sequence: active
 	UpdatesDerrickCount:
 	Building:
+		Footprint: xx xx xx
+		Dimensions: 2,3
 		RequiresBaseProvider: false
-		TerrainTypes: Ore
-		AllowInvalidPlacement: true # TODO allow placement only on ressources
+		TerrainTypes: Clear, Road, Crater, Grass, Ore
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml]: Can you please revert that?

Comment (GraionDilach) [reply mods/hv/rules/buildings.yaml]: Why? Won't this be required for longterm?

I thought the idea is to allow placing the miner building upon single-cell patches of resource as well - in which case the other cells will need to remain plain buildable cells to fit the extractor in and those cells will also need to satisfy the building's own validity check.

The GrantConditionOnTerrain check prevents the miner already to deploy to positions where no resource is under it, this just gives more room for building placement (assuming the CanPlaceBuilding issue is fixed).

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml]: You are right, the miner is not 1x1.

File: mods/hv/rules/buildings.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/hv/rules/buildings.yaml]: #17838

**Pull request merged**